contrib/conan/recipes/outcome/conanfile.py

```diff
@@ -0,0 +1,21 @@
+# from https://github.com/ned14/outcome/blob/develop/conan/conanfile.py
+from conans import ConanFile, tools
+
+class OutcomeConan(ConanFile):
+    name = "Outcome"
+    version = "3dae433e"
+    license = "Apache-2.0"
+    description = "Provides very lightweight outcome<T> and result<T>"
+    repo_url = "https://github.com/ned14/outcome"
+
+    def source(self):
+        file_url = "https://raw.githubusercontent.com/ned14/outcome/3dae433e279090291a2fc21d3c54367f08f131f8/"
```

Comment:
you could change this line to `file_url = "https://raw.githubusercontent.com/ned14/outcome/{}/".format(self.version)`. This way the version won't accidentally get out of sync.